pkgs/applications/audio/musescore/darwin.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
   pname = "musescore-darwin";
   version = concatStringsSep "." versionComponents;
 
+  # The disk image contains the .app and a symlink to /Applications.
+  setSourceRoot = ''
+    sourceRoot=$(echo *.app)
```

Comment:
What about `sourceRoot="${appName}.app";` instead of setSourceRoot?

Reply:
Either seems fine me.